.github/workflows/daily-deploy.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Deploy
         run: |
-          if npm run deploy |& grep -q "'${{ matrix.project.event }}'"; then
+          if npm run deploy | grep -q "'${{ matrix.project.event }}'"; then
```

Comment:
yes, it doesn't work with "&"

Reply:
I mean ok, it fails with & but if we remove it, maybe this part of the ci become useless because it wil always pass, even if the deployment itself fails

Reply:
I tested the command and it also returns an error when there is a problem on npm run deploy.

"|" redirects the standard output of npm run deploy
"|&" redirects both the standard output and standard error output of npm run deploy

Reply:
nice. 
So now, is there a point for us to keep the grep ?

Reply:
I don't think it's mandatory. It allows you to check the event generated, no more.

Reply:
so maybe we should remove it